### PR TITLE
CSP-165-Outer-UpdateProviderLocationDetails

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/InnerApi/Requests/ProviderLocationUpdateRequest.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using System;
+
+namespace SFA.DAS.RoatpCourseManagement.InnerApi.Requests
+{
+    public class ProviderLocationUpdateRequest : IPutApiRequest<ProviderLocationUpdateModel>
+    {
+        public int Ukprn { get; }
+        public Guid Id { get; }
+        public string UserId { get; set; }
+        public string PutUrl => $"providers/{Ukprn}/locations/{Id}/";
```
Comment (MarkFCain) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/InnerApi/Requests/ProviderLocationUpdateRequest.cs:11]: this needs some coverasge, to check the model mapping and the PutUrl

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Application/Locations/Commands/UpdateProviderLocationDetails/UpdateProviderLocationDetailsCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            var updateProviderLocation = new ProviderLocationUpdateModel
+            {
+                Ukprn = command.Ukprn,
+                Id = command.Id,
+                UserId = command.UserId,
+                LocationName = command.LocationName,
+                Website = command.Website,
+                Email = command.Email,
+                Phone = command.Phone
+            };
```
Comment (MarkFCain) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Application/Locations/Commands/UpdateProviderLocationDetails/UpdateProviderLocationDetailsCommandHandler.cs:29]: You could have the mapping of the command within the ProviderLocationUpdateModel.
I'll leave the decision to do that to you

**Review by MarkFCain — DISMISSED** [review]
A request for extra coverage and an optional suggestion about model mapping

**Review by DevenKShah — APPROVED** [review]
Coverage is missing, but since the story is tested, I am approving it.

**Pull request merged**